index.html

```diff
@@ -71,8 +74,289 @@ <h1>I’m Noel Bryant from Kenya. Glad to see you!</h1>
+                  <img src="assets/images/social-media-icons/linkedin.svg"
+                    /></a>
+              </div>
+
+              <div class="medium">
+                <a href="index.html">
+                  <img src="assets/images/social-media-icons/medium.svg" />
+
+                </a>
+              </div>
+
+              <div class="github">
+                <a href="https://github.com/NoelLincoln">
+                  <img src="assets/images/social-media-icons/github.svg" />
+                </a>
+              </div>
+              <div class="peacesign">
+                <a href="index.html">
+                  <img src="assets/images/social-media-icons/peacesign.svg" />
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="resume-button">
+            <div class="resume">
+              <a
+                href="https://drive.google.com/file/d/1iD2S5PJAGP1TI_19C9GD2Zo4FpeSRO1N/view?usp=sharing">Get
+                My resume</a>
+            </div>
+          </div>
+
+        </div>
+        <div class="skill-list">
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn" id="languages">
+              <span class="languages">Languages</span>
+              <span class="arrow right"></span>
+            </label>
+
+            <ul class="dropdown-content" role="menu">
+              <li><img src="assets/images/javascript.png" alt="javascript
+                  image"/><a href="#">Javascript</a></li>
+              <li><img src="assets/images/html.png" alt="html
+                  image"/><a href="#">HTML</a></li>
+              <li><img src="assets/images/javascript.png" alt="javascript
+                  image"/><a href="#">CSS</a></li>
+            </ul>
+          </div>
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn">
+              <span class="languages">Frameworks</span>
+              <span class="arrow right"></span>
+            </label>
+
+            <ul class="dropdown-content-frameworks" role="menu">
+              <li><a href="#">React</a></li>
+              <li><a href="#">Angular</a></li>
+              <li><a href="#">Vue</a></li>
+              <li><a href="#">Svelte</a></li>
+            </ul>
+          </div>
+          <div class="dropdown">
+            <input type="checkbox" id="dropdown" />
+            <label for="dropdown" class="dropdown-btn">
+              <span class="languages">Skills</span>
+              <span class="arrow right"></span>
+            </label>
+
+            <ul class="dropdown-content-skills" role="menu">
+              <li><a href="#">React</a></li>
+              <li><a href="#">Angular</a></li>
+              <li><a href="#">Vue</a></li>
+              <li><a href="#">Svelte</a></li>
+            </ul>
+          </div>
+        </div>
+
```

Comment:
- [ ] Your technology categories such as `languages, frameworks and skills` should be wrapped in `ul` and `li` tags. It is important to make use of semantic list item tags to display the list of technologies. :+1: